content/docs/user-guide/visualizing-plots.md

```diff
@@ -112,7 +113,11 @@ names in the `train` array below:
 }
 ```
 
-### Example: Tabular data
+[dvc template anchors]: /doc/command-reference/plots/templates#custom-templates
+
+<details>
+
+### Example: Tabular data (CSV, TSV)
 
 We'll use tabular metrics file `logs.csv` for these examples:
```

Comment:
One problem in the guide: these examples are repeated in https://dvc.org/doc/command-reference/plots/show. We should have them in a single place. Probably the cmd ref (casually link there if needed).

Reply:
I'm leaving them here unless we have a strong reason not to have them. For this guide especially, example visualizations help a lot, and we have almost none in the guide as is. Also, people can use plots with VS Code or Studio and never touch the plots commands, so I don't think it's sufficient to have them there.

Reply:
Remove them from the command ref then? (Link to here)